pkg/config/config.go

```diff
@@ -211,6 +213,16 @@ func Scan(opts ...Option) (c *Config, err error) {
 
 		yaml.Unmarshal(body, c)               //nolint:errcheck
 		yaml.Unmarshal(body, &c.originalData) //nolint:errcheck
+
+		if exists, header := HasHeader(string(body), ""); c.IsValid() || exists {
```

Comment:
`c.IsValid()` has no effect here. If `exists` is true, then we enter the block, if it's not then we may enter the block but the block will do nothing. Not sure why `IsValid` is called here.

Reply:
right, good catch, totally missed it!